target/linux/ramips/image/mt7620.mk

```diff
@@ -127,52 +127,52 @@ define Device/bdcom_wap2100-sk
 endef
 TARGET_DEVICES += bdcom_wap2100-sk
 
-define Device/bocco
+define Device/yukai_bocco
   DTS := BOCCO
   DEVICE_TITLE := YUKAI Engineering BOCCO
   DEVICE_PACKAGES := kmod-sound-core kmod-sound-mt7620 kmod-i2c-ralink
   SUPPORTED_DEVICES += planex,cs-qr10
 endef
-TARGET_DEVICES += bocco
+TARGET_DEVICES += yukai_bocco
 
-define Device/c108
+define Device/hnet_c108
```

Comment:
However when doing these renamings you'll need to add SUPPORTED_DEVICES as the default one in target/linux/ramips/image/Makefile may not match board name in base-files/lib/ramips.sh.

Reply:
Ah, I now realize that I had a wrong understanding of how the board name for upgrade was derived. I thought it was based on the compatible, as in ath79 the compatible and device name match each other (except , vs _).
I will have to think about the best way to address this.